isis/src/base/apps/spiceinit/tsts/shapemodel/Makefile

```diff
@@ -1,25 +0,0 @@
-# Test spiceinit on a file which uses a shapemodel with the Renderer, etc specified in an 
```

Comment:
This test was not ported over yet because we didn't have a way to test different preference files in the gtests.

Reply:
reverted